### PR TITLE
docs(android/app): Update tablet screenshots

### PR DESCRIPTION
**Pull request by darcywong00**
Follows #10691 for updating tablet screenshots in-app content for #10629

@keymanapp-test-bot skip

### CODE REVIEW

File: android/help/android_images/dist-install1-at.png
Comment (mcdurdin) [inline android/help/android_images/dist-install1-at.png:1]: I didn't realise the banner text was such a terrible low contrast. That's a bug that we need to fix.

**Review by mcdurdin — APPROVED** [review]
LGTM although there is an underlying bug which we do need to fix which will mean redoing a couple of the images

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.271-beta](https://keyman.com/downloads/releases/beta/17.0.271)

File: android/help/android_images/dist-install1-at.png
Comment (darcywong00) [reply android/help/android_images/dist-install1-at.png:1]: Banner addressed in #10760 and screens updated in #10796

**Pull request merged**